tests/0105-transactions_mock.c

```diff
@@ -3044,5 +3170,9 @@ int main_0105_transactions_mock(int argc, char **argv) {
 
         do_test_txn_coordinator_null_not_fatal();
 
+        do_test_txn_resumable_calls_timeout(rd_true);
+
+        do_test_txn_resumable_calls_timeout(rd_false);
```

Comment:
Do I add also the tests from #4016 about misuse for commit and abort and the misuse with concurrent call from a different thread? (do_test_txn_unexpected_operations and do_test_txn_concurrent_operations)